io.catenax.essincident/3.0.0/EssIncident.ttl

```diff
@@ -0,0 +1,334 @@
+   samm:preferredName "Incident Category"@en ;
+   samm:description "Environmental and social standards related incident category according to Supply Chain Due Diligence Act"@en ;
+   samm:see <https://www.gesetze-im-internet.de/lksg/> ;
+   samm:characteristic :IncidentCategoryCharacteristic ;
+   samm:exampleValue "Social" .
+
+:incidentSubcategory a samm:Property ;
+   samm:preferredName "Incident Subcategory"@en ;
+   samm:description "Subcategory of an incident Category"@en ;
+   samm:characteristic :IncidentSubcategories ;
+   samm:exampleValue "Child labour" .
+
+:incidentSubject a samm:Property ;
+   samm:preferredName "Incident Subject"@en ;
+   samm:description "Title/ subject of an incident in the context of ESS (Environmental and Social Standards)"@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "Child labour on rubber producer Farm A" .
+
+:incidentExternalSubject a samm:Property ;
+   samm:preferredName "External Subject"@en ;
+   samm:description "Replaces Incident Subject for external communication"@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "Child labour on a rubber producer farm" .
+
+:incidentDescription a samm:Property ;
+   samm:preferredName "Incident Description"@en ;
+   samm:description "Full text description of an incident in the context of ESS (Environmental and Social Standards)"@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "Child labour at production site of the rubber producer Farm A in Brazil" .
+
+:incidentExternalNotes a samm:Property ;
+   samm:preferredName "External Notes"@en ;
+   samm:description "Incident description for external communication"@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "Child labour at production site of a rubber producer in Brazil" .
+
+:incidentAttachment a samm:Property ;
+   samm:preferredName "Incident Attachment"@en ;
+   samm:description "Picture(s) about the reported incident in the context of ESS (Environmental and Social Standards)"@en ;
+   samm:characteristic samm-c:ResourcePath .
+
+:systemDate a samm:Property ;
+   samm:preferredName "System Date"@en ;
+   samm:description "System created timestamp when the incident in the context of ESS (Environmental and Social Standards) was issued and saved"@en ;
+   samm:characteristic samm-c:Timestamp ;
+   samm:exampleValue "2022-08-31T23:22:12Z"^^xsd:dateTime .
+
+:incidentDate a samm:Property ;
+   samm:preferredName "Date of Incident"@en ;
+   samm:description "Date and time information when an incident in the context of ESS (Environmental and Social Standards) occurred"@en ;
+   samm:characteristic samm-c:Timestamp ;
+   samm:exampleValue "2022-08-31T00:00:00Z"^^xsd:dateTime .
+
+:incidentId a samm:Property ;
+   samm:preferredName "Incident ID"@en ;
+   samm:description "Unique identifier for an incident in the context of ESS (Environmental and Social Standards)"@en ;
+   samm:characteristic ext-uuid:Uuidv4Characteristic ;
```

Comment:
If you want you can directly link trait instead of Characteristic. It will add the usage of constraint automatically at the same time.
example https://github.com/eclipse-tractusx/sldt-semantic-models/pull/600/files#diff-1c4db0c94a76620747c6f8dcc2656113dcb1aac2a932d9b964e6a514a4bb8824R43

Reply:
I linked shared trait (symbol T in aspect modeler) with my Property (symbol P in aspect modeler) and then tried to link the shared trait (symbol T in aspect modeler) with my Characteristic (symbol C in aspect modeler). Received error message "cannot connect external reference". 
I could not open the model example itemstock 2.0.0 you linked above. Can you advise?

Reply:
just add this as a characteristic in all three properties.
`samm:characteristic ext-uuid:UuidV4Trait;`
Let me know if you still face problem.

Reply:
sorry, I was ill thius week so I could not get back to you earlier. That was exactly what I did in the latest version of the model (link the characteristic to the properties), but I am not sure if this is what you expected when you asked to add the trait to the properties.
it looks like this: see screenshot
![screenshot](https://github.com/eclipse-tractusx/sldt-semantic-models/assets/103646525/36968d26-434c-4573-9fb7-fc76c56b4601)
Let me know if this is not what you meant. Thanks

Reply:
Thanks for the hint, got it now and committed the changed with referencec to trait.